blog/content/post/2025-11-03-nis2-digital-sovereignty-with-zarhus.md

```diff
@@ -0,0 +1,327 @@
+cybersecurity risks in supply chains. Key stipulations include implementing
+comprehensive supply chain risk management plans, assessing vulnerabilities in
+suppliers and service providers, and enforcing strict cryptographic policies to
+protect sensitive data.
+
+[ENISA Executive Director, Juhan Lepassaar
+stated](https://web.archive.org/web/20251003154059/https://enisa.europa.eu/news/etl-2025-eu-consistently-targeted-by-diverse-yet-convergent-threat-groups):
+
+> Systems and services that we rely on in our daily lives are intertwined, so a
+> disruption on one end can have a ripple effect across the supply chain. This
+> is connected to a surge in abuse of cyber dependencies by threat actors that
+> can amplify the impact of cyberattacks.
+
+Organizations are required not only to adopt technical and operational
+safeguards but also to coordinate with suppliers for consistent risk evaluation
+and response. Through these mandates, NIS2 aims to close security gaps that
+threat actors have long exploited in interconnected digital supply
+environments.
+
+But how can effective coordination and comprehensive risk evaluation be
+achieved when your organization relies on black-box devices, without the
+ability to transparently validate machine integrity? What strategies should be
+employed when your vendor discontinues firmware support for critical hardware?
+How can you guarantee that the foundational firmware truly serves **your**
+operational requirements rather than external interests?
+
+This article takes a deep dive into these challenges and illustrates how
+adopting [Zarhus](https://docs.zarhus.com/) enables precise [control over your
+hardware’s low-level
+functions](https://docs.dasharo.com/osf-trivia-list/dasharo/#future-work),
+enhancing [security,
+reliability](https://docs.dasharo.com/osf-trivia-list/dasharo/#dasharo-professional-support),
+and [long-term
+maintainability](https://docs.dasharo.com/osf-trivia-list/dasharo/#dasharo-long-term-maintenance).
+
+![Zarhus logo](/img/zarhus-logo-new.png)
+
+## The Challenge of Firmware Security
+
+Let's simplify the requirements from [Article
+21.2](https://web.archive.org/web/20250916222654/https://eur-lex.europa.eu/legal-content/EN/TXT/HTML/?uri=CELEX%3A32022L2555)
+to address these specific points:
+
+- Supply Chain Security
+  - Entities must implement appropriate technical, operational, and
+      organizational measures to manage cybersecurity risks related to supply
+      chains.
+  - Assess vulnerabilities of direct suppliers and service providers,
+      including their secure development practices.
+  - Take coordinated security risk assessments into account to ensure
+      comprehensive supply chain risk management.
+- Cryptography Usage
+  - Establish policies and procedures for the proper use of cryptography and
+      encryption.
+  - Cryptographic measures must be appropriate, proportionate, and aligned
+      with overall cybersecurity risk management.
```

Comment:
This is very generic and subject to interpretation. Our goal in this post should be justification of our interpretation, why we think given approach address given point.

We should also clearly state that such interpreation may be different depending on threat model.

Reply:
fixed: https://github.com/3mdeb/news-and-ideas/compare/aa6b917925579ce12e9b5440610792e79ab908ae..3d442a6beb3dd9fcfd12ac211854fc7065a6ca67#diff-7945a7d57c8b59ce3c93bc1320a0777450daf776f04eaeebd44545ada6b4545aR118-R137